src/secondAssignment_14March2023/FindDestinationCity.java

```diff
@@ -0,0 +1,22 @@
+package secondAssignment_14March2023;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+
+	public static void main(String[] args) {
+		String input[][] = { { "London", "Newyork" }, { "Newyork", "Lima" }, { "Lima", "Sao Paulo" } };
+		// String input[][] = { { "B", "C" }, { "D", "B" }, { "C", "A" } };
+		ArrayList<String> AL = new ArrayList<>();
+		for (int i = 0; i < input.length; i++) {
```

Comment:
create a new function and use it in main

Reply:
Done